test/test_helper.rb

```diff
@@ -1,6 +1,17 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "sorbet-runtime"
+
+if ENV["COVERAGE"]
+  require "simplecov"
+
+  SimpleCov.start do
+    T.bind(self, SimpleCov::Configuration)
+    enable_coverage :branch
```

Comment:
Out of curiosity, what does this do?

Reply:
It checks condition coverage too (versus just line coverage), which is particularly useful for inline conditions like `foo if bar?`

Reference: https://github.com/simplecov-ruby/simplecov?tab=readme-ov-file#branch-coverage-ruby--25